packages/flutter/lib/src/material/about.dart

```diff
@@ -455,6 +465,7 @@ class _LicensePageState extends State<LicensePage> {
     assert(debugCheckHasMaterialLocalizations(context));
     final String name = widget.applicationName ?? _defaultApplicationName(context);
     final String version = widget.applicationVersion ?? _defaultApplicationVersion(context);
+    final Widget icon = widget.applicationIcon ?? _defaultApplicationIcon(context);
```

Comment:
Thanks for being thorough here, even though `_defaultApplicationIcon` has not been implemented yet